site/layouts/pages/hub-login.html

```diff
@@ -0,0 +1,122 @@
+<!doctype html>
+<html lang="{{ $.Site.Language.Lang }}">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
+    <title>{{ $.Site.Title }}</title>
+    <link rel="stylesheet" href="{{.Site.BaseURL}}css/styleguide-hub.css" />
+
+    <link rel="apple-touch-icon" sizes="180x180" href="{{.Site.BaseURL}}apple-touch-icon.png?2020">
+    <link rel="icon" type="image/png" sizes="32x32" href="{{.Site.BaseURL}}favicon-32x32.png?2020">
+    <link rel="icon" type="image/png" sizes="16x16" href="{{.Site.BaseURL}}favicon-16x16.png?2020">
+    <link rel="icon shortcut" type="image/x-icon" href="{{.Site.BaseURL}}favicon.ico?2020">
+    <link rel="icon" type="image/x-icon" href="{{.Site.BaseURL}}favicon.ico?2020">
+    <link rel="manifest" href="{{.Site.BaseURL}}site.webmanifest?2020">
+    <link rel="mask-icon" href="{{.Site.BaseURL}}safari-pinned-tab.svg?2020" color="#e11a81">
+    <meta name="msapplication-TileColor" content="#da532c">
+    <meta name="theme-color" content="#ffffff">
+</head>
+
+<body class="styleguide hub">
+  <a class="back-to-design-system" href="{{ .Site.BaseURL }}">Revenir au Design System</a>
+  <a class="go-to-content screen-reader screen-reader-focusable" href="#main">
+    Aller au contenu
+  </a>
+
+  <header class="header--hub" role="banner">
+    <div class="logo">
+      <img src="{{.Site.BaseURL}}images/hub-2020/logo-pw2020.svg" width="200" height="140" alt="" />
+    </div>
+  </header>
+
+  <main class="main--hub" id="main" tabindex="-1">
```

Comment:
```suggestion
  <main class="main--hub" id="main">
```

Cette valeur de tabindex le rends cliquable de manière un peu étrange lorsqu'on clique sur la page. A priori on ne devrait pas avoir besoin de le rendre *focusable*.

![image](https://user-images.githubusercontent.com/132/93014846-cd916e00-f5b4-11ea-8bb7-1cd2ba6a6121.png)